edx/edx-ora2
# Make submission feedback full-width

**TL;DR -** in the "Your Grade" section of ORA, make the feedback section full-width to make better use of screen space (as below)

JIRA: [EDUCATOR-5805](https://openedx.atlassian.net/browse/EDUCATOR-5805)

**TODO**
- [x] A11y signoff
- [x] UX signoff
- [x] Mobile team signoff
- [x] Product signoff

<img width="922" alt="Screen Shot 2021-06-02 at 16 16 57" src="https://user-images.githubusercontent.com/12944465/120551058-9d4c6280-c3c3-11eb-9ec8-b87112470f8a.png">

**Justification**

We use an older grid system ([Neat v1.7.0](https://github.com/thoughtbot/neat)) for "responsive" layout. Neat enforces a grid system, configured for different screen sizes. In the "Feedback" section of "Your Grade", it is configured for the breakpoints below:
small / mobile | medium | large
--- | --- | ---
1 column | 2 columns | 3 columns

Since it is older (and self-suggests replacement with newer systems like flexbox/grid), it doesn't dynamically fill the space when it has fewer elements than columns, leading to wasted screen space (as in screenshot below). This is exacerbated by recent work which allowed for much longer (10,000 character) feedback responses.

We could fix with flexbox, allowing fewer elements to greedily fill the space, or just max-width everything. In the spirit of simplicity I picked the latter.

Shouldn't affect mobile view or LTR/RTL layout. Does introduce a minor readability issue of longer lines but am consulting with @wittjeff to see if it's an issue or not. Also need to run by UX.

<img width="920" alt="Screen Shot 2021-06-02 at 12 28 12" src="https://user-images.githubusercontent.com/12944465/120550941-7beb7680-c3c3-11eb-972a-8e4a4b85b445.png">

**Developer Checklist**


**Testing Instructions**

1. Create or use an ORA (I suggest a staff/self-graded only or peer review with 2 reviewers for speed)
2. Submit a response to the ORA
3. As staff/other users grade the response with feedback for response and rubric criteria. Vary length of feedback text.
4. As the original user, continue to the "Your Grade" section and expand to view grade and feedback.
5. Verify that feedback takes full width of ORA container
6. Stylistic/functionality smoke test, confirm that nothing else looks/acts strangely

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
